src/ovp_pipeline/commands/_ui_renderers.py

```
@@ -1408,13 +1416,336 @@ def _render_markdown_note(
+       page renders correctly even if the ``type:`` key was lost.
+
+    Files outside all four keep the existing full-scaffold
+    behaviour so evergreens, deep-dives, and atlas pages render
+    unchanged.
+    """
+    normalised = (relative_path or "").replace("\\", "/")
+    if any(normalised.startswith(prefix) for prefix in _THIN_NOTE_PATH_PREFIXES):
+        return True
+    try:
+        frontmatter, _ = _parse_frontmatter(markdown)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)